docker-compose.test.yml

```diff
@@ -48,9 +47,18 @@ services:
       - PRIORITY_ACO_IDS=A9990,A9991,A9992,A9993,A9994
     volumes:
       - .:/go/src/github.com/CMSgov/bcda-app
+  db-test:
```

Comment:
Should this be called `db-unit-test` to be explicit, so that folks aren't confused when running postman/integration tests?  This DB is only used for unit tests, and not used for any other tests, and I think we need to make that clear.

Reply:
👍